www/js/lib/uiUtil.js

```diff
@@ -375,18 +394,18 @@ define(rqDef, function() {
         // Show any specific help for selected contentTheme
         var help = document.getElementById(theme + '-help');
         if (help) help.style.display = 'block';
-        
+        var frame = container.classList ? container : container.document.documentElement;
```

Comment:
If articleContainer is an iframe, it will already have a `classList`. If it is a Window, we have to apply theme classes to the documentElement (i.e. the `<html>` node).